app/src/main/java/com/nitrr/ecell/esummit/ecellapp/activities/HomeActivity.java

```diff
@@ -43,17 +38,21 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     public void initializeList() {
+        HomeRVData data4 = new HomeRVData();
+        data4.setName("BQuiz");
+        data4.setImage(R.drawable.ic_google);
+        homeRVDataList.add(data4);
         HomeRVData data = new HomeRVData();
-        data.setName("Example");
-        data.setImage(R.drawable.ic_google);
+        data.setName("Events");
+        data.setImage(R.drawable.ic_events);
         homeRVDataList.add(data);
         HomeRVData data2 = new HomeRVData();
-        data2.setName("Example");
-        data2.setImage(R.drawable.ic_google);
+        data2.setName("ESummit");
```

Comment:
Make a constructor to set everything at once. Don't increasing LoC.

Reply:
Done!